pkg/promclient/promclient.go

```diff
@@ -607,6 +607,34 @@ func (c *Client) AlertmanagerAlerts(ctx context.Context, base *url.URL) ([]*mode
 	return v.Data, nil
 }
 
+// BuildInfo returns Prometheus version from /api/v1/status/buildinfo Prometheus endpoint.
+func (c *Client) BuildInfo(ctx context.Context, base *url.URL) (string, error) {
+	u := *base
+	u.Path = path.Join(u.Path, "/api/v1/status/buildinfo")
+
+	level.Debug(c.logger).Log("msg", "querying instant", "url", u.String())
+
+	span, ctx := tracing.StartSpan(ctx, "/prom_buildinfo HTTP[client]")
+	defer span.Finish()
+
+	body, _, err := c.req2xx(ctx, &u, http.MethodGet)
+	if err != nil {
+		return "", err
+	}
+
+	var b struct {
+		Data struct {
+			Version string `json:"version"`
+		} `json:"data"`
+	}
+
+	if err = json.Unmarshal(body, &b); err != nil {
+		return "", errors.Wrap(err, "unmarshal build info API response")
+	}
+
+	return b.Data.Version, nil
```

Comment:
Okay so this function just wants to return the version of the Prometheus instead of returning all the buildInfo. So maybe let's rename it to `BuildVersion`? WDYT